code/app/src/main/java/com/adobe/marketing/edge/identity/app/EdgeIdentityApplication.kt

```diff
@@ -16,11 +16,12 @@ import com.adobe.marketing.mobile.Assurance
 import com.adobe.marketing.mobile.Edge
 import com.adobe.marketing.mobile.LoggingMode
 import com.adobe.marketing.mobile.MobileCore
+import com.adobe.marketing.mobile.edge.consent.Consent
 import com.adobe.marketing.mobile.edge.identity.Identity
 
 class EdgeIdentityApplication : Application() {
-    // Add your Launch Environment ID to configure the SDK from your Launch property
-    private var LAUNCH_ENVIRONMENT_ID: String = ""
+    // Add your app environment id to configure the SDK from your Data Collection mobile property
```

Comment:
@timkimadobe we can update this to the same as iOS when you update the test app in the next PR.

// TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
   
private var ENVIRONMENT_FILE_ID: String = "yourAppId"

MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID)

Reply:
Updated to the new naming style!